### PR TITLE
New initial "Flutter Test (Bazel)" run configuration boiler plate code

### PR DESCRIPTION
**Pull request by jwren**
New initial "Flutter Test (Bazel)" run configuration boiler plate code; currently this new configuration is commented out in the plugin.xml and in a work-in-progress state; currently running a test works, debugging does not; by landing this initial CL follow on CLs can be smaller and incremental.

### CODE REVIEW

**Comment by devoncarew** [comment]
In order to get travis happy, when you change the plugin.xml file, you should apply your actual edits to `resources/META-INF/plugin_template.xml`, and then run `bun/plugin generate`.

File: src/io/flutter/run/bazelTest/BazelTestFields.java [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+  }
+
+  //@Nullable
+  //public String getAdditionalArgs() {
+  //  return additionalArgs;
+  //}
+  //
+  //public void setAdditionalArgs(@Nullable String additionalArgs) {
+  //  this.additionalArgs = additionalArgs;
+  //}
```
Comment (pq) [inline src/io/flutter/run/bazelTest/BazelTestFields.java]: Can we remove these?  (Or just add them in?)

File: src/io/flutter/run/bazelTest/BazelTestFields.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    final DartSdk sdk = DartPlugin.getDartSdk(project);
+    if (sdk == null) {
+      throw new RuntimeConfigurationError(FlutterBundle.message("dart.sdk.is.not.configured"),
+                                          () -> DartConfigurable.openDartSettings(project));
+    }
+
+    //final MainFile.Result main = MainFile.verify(entryFile, project);
+    //if (!main.canLaunch()) {
+    //  throw new RuntimeConfigurationError(main.getError());
+    //}
```
Comment (pq) [inline src/io/flutter/run/bazelTest/BazelTestFields.java]: Is this still WIP or can we safely remove?

File: src/io/flutter/run/bazelTest/BazelTestFields.java [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    //// More user-specified args.
+    //while (argumentsTokenizer.hasMoreTokens()) {
+    //  commandLine.addParameter(argumentsTokenizer.nextToken());
+    //}
+    //
+    //// Send in the deviceId.
+    //if (device != null) {
+    //  commandLine.addParameter("-d");
+    //  commandLine.addParameter(device.deviceId());
+    //}
```
Comment (pq) [inline src/io/flutter/run/bazelTest/BazelTestFields.java]: Similarly here with all of this.  Maybe add a TODO or something if it needs to stay?

File: src/io/flutter/run/bazelTest/BazelTestFields.java [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+      throw new RuntimeConfigurationError(FlutterBundle.message("dart.sdk.is.not.configured"),
+                                          () -> DartConfigurable.openDartSettings(project));
+    }
+
+    //final MainFile.Result main = MainFile.verify(entryFile, project);
+    //if (!main.canLaunch()) {
+    //  throw new RuntimeConfigurationError(main.getError());
+    //}
+
+    // check launcher script
```
Comment (pq) [inline src/io/flutter/run/bazelTest/BazelTestFields.java]: Nit: comments elsewhere start with a capital. 

=> // Check ...

File: src/io/flutter/run/bazelTest/BazelTestFields.java (hunk 2 above)
Comment (jwren) [reply src/io/flutter/run/bazelTest/BazelTestFields.java]: I don't know yet if we should remove or add them in.  Added TODOs.

File: src/io/flutter/run/bazelTest/BazelTestFields.java (hunk 1 above)
Comment (jwren) [reply src/io/flutter/run/bazelTest/BazelTestFields.java]: I don't know yet if we should remove or add them in.  Added TODOs.

File: src/io/flutter/run/bazelTest/BazelTestFields.java (hunk 4 above)
Comment (jwren) [reply src/io/flutter/run/bazelTest/BazelTestFields.java]: Fixed here and the other locations in this file.  I'll fix the occurrences in BazelFields.java in a follow up PR

**Pull request merged**